kube/client.go

```diff
@@ -36,7 +36,6 @@ var execCommand = exec.Command
 //todo(catalin-ilea) Add core/v1/Secret when we plug in strongbox
 var pruneWhitelist = []string{
 	"apps/v1/DaemonSet",
-	"apps/v1beta1/DaemonSet",
```

Comment:
I remember adding that because people were still using `v1beta1/DaemonSet`. Is that a way to force people to update their manifests?

Reply:
We can say that we no longer support pruning of deprecated resources.